shared/AppInsightsCommon/src/Interfaces/IConfig.ts

```diff
@@ -311,6 +311,22 @@ export interface IConfig {
      */
     namePrefix?: string;
 
+    /**
+     * @description An optional value that will be used as name postfix for session cookie name.
+     * @type {string}
+     * @memberof IConfig
+     * @defaultValue null
+     */
+    sessionCookiePostfix?: string;
```

Comment:
Add details about the fallback logic and also update the READMD.md with the details, so that users know about how this works.

I've also started adding some indication on when any new configs where added (I think mostly the internal code)